### ISSUE TITLE
Remove `APIKeyMissingError`

### ISSUE THREAD

**Issue by camilamaia** [scanapi/scanapi#165]
Remove `APIKeyMissingError`. We are not using it anymore, we are using `MissingMandatoryKeyError` instead. 

We only need to delete [it](https://github.com/scanapi/scanapi/blob/master/scanapi/errors.py#L15-L20)

```
class APIKeyMissingError(MalformedSpecError):
    """Raised when 'api' key is not specified at root scope in the API spec"""

    def __init__(self, *args):
        message = "Missing api 'key' at root scope in the API spec"
        super(APIKeyMissingError, self).__init__(message, *args)
```

**Comment by vinigfer** [comment]
@camilamaia Could you assign this for me?
Implemented it on this Pull Request https://github.com/scanapi/scanapi/pull/218

**Comment by camilamaia** [comment]
closed by #218
